src/frontend/planner_test/tests/testdata/output/range_scan.yaml

```diff
@@ -35,35 +35,39 @@
   batch_error: |
     Expr error
 
-    Caused by:
-      Division by zero
+    Caused by these errors (recent errors listed first):
+      1: error while evaluating expression `general_div(1, 0)`
+      2: Division by zero
 - before:
   - create_table_and_mv
   sql: |
     SELECT * FROM orders_count_by_user WHERE user_id = 2147483647 + 1
   batch_error: |
     Expr error
 
-    Caused by:
-      Numeric out of range
+    Caused by these errors (recent errors listed first):
+      1: error while evaluating expression `general_add(2147483647, 1)`
+      2: Numeric out of range
 - before:
   - create_table_and_mv
   sql: |
     SELECT * FROM orders_count_by_user WHERE user_id = 'a'
   batch_error: |
     Expr error
 
-    Caused by:
-      Parse error: bigint invalid digit found in string
+    Caused by these errors (recent errors listed first):
+      1: error while evaluating expression `str_parse(a)`
```

Comment:
Should this be `str_parse('a')`?

Reply:
This is because I call `ToText` internally to format them, which shares the same behavior as `xx::varchar` in SQL and may not be consistent with the literal values. 😕 Not sure if there's some existing utility to do this. cc @xiangjinwu

Reply:
There is the recently added `quote_nullable` #16807. Note that it may be necessary to use explicit cast after argument literal to disambiguate when the function is overloaded:
```
test=# select md5('\xabcd'::varchar);
               md5                
----------------------------------
 7e9d4239869ab046747b53ef3a8cdd1e
(1 row)

test=# select md5('\xabcd'::bytea);
               md5                
----------------------------------
 7838496fd0586421bbb500bb6f472f13
(1 row)

test=# select md5('\xabcd'); -- when ambiguous, varchar is preferred
               md5                
----------------------------------
 7e9d4239869ab046747b53ef3a8cdd1e
(1 row)
```

Reply:
> There is the recently added `quote_nullable` #16807.

Good point.

> Note that it may be necessary to use explicit cast after argument literal to disambiguate when the function is overloaded:

We show the backend (Rust) function name in the error message. So I guess we can distinguish between the overloads there.